platform/spire/src/seq.py

```diff
@@ -59,6 +59,14 @@ def sequence_supervisor(ops: command.Operations, skip_verify_keygateway: bool=Fa
         if node.kind == 'supervisor':
             ops.add_subcommand(infra.infra_sync, node.hostname)
 
+@command.wrapseq
+def sequence_redeploy_config(ops: command.Operations) -> None:
+    "redeploy a cluster configuration to a running cluster"
+    # push new config to the keyserver
+    setup.redeploy_keyserver(ops)
+    # push new config to each keyclient and restart
+    setup.redeploy_keyclients(ops)
```

Comment:
Since the `seq` file is mostly for things that are composites of other commands, I think the "correct" thing to do is either:
 - to change redeploy_keyserver and redeploy_keyclients to be commands and use add_subcommand
 - OR to move this command to setup.py
But I think this PR has been held up long enough, and it's not like spire is very consistent as things are, so I don't think it's worth making this change.